website/docs/r/cosmosdb_account.html.markdown

```diff
@@ -106,6 +106,16 @@ The following arguments are supported:
 
 * `enable_multiple_write_locations` - (Optional) Enable multi-master support for this Cosmos DB account.
 
+* `key_based_meta_write_access_enabled` - (Optional) Is write operations on metadata resources (databases, containers, throughput) via account keys enabled? Defaults to `true`.
```

Comment:
this makes little sense, by account keys does it mean the access keys? are there are ways you can write?
```suggestion
* `access_key_metadata_writes_enabled` - (Optional) Is write operations on metadata resources (databases, containers, throughput) via account keys enabled? Defaults to `true`.
```